doc/Learn/Maui/MauiOverview.md

```diff
@@ -136,6 +160,10 @@ Add a reference in the existing class library to [Uno.Extensions.Maui.WinUI](htt
     </Project>
     ```
 
+***
+
+### Maui Class Library
+
 1. **Maui class library**
```

Comment:
We now have a header and this right after both saying "Maui class library"

Reply:
I've changed the Header to **Add MAUI Application** and the following sub-header left as it was as 1. **MAUI Class Library**

![image](https://github.com/unoplatform/uno.extensions/assets/114886335/1e9aebed-a935-4126-b04c-67458c6f3cd9)

Is that better?

Reply:
Noticed "Application" is missing a "p". Will adjust.